packages/context/src/oidcContext/AuthenticationContext.tsx

```diff
@@ -9,6 +9,7 @@ export type oidcContext = {
   events: UserManagerEvents;
   authenticating: ComponentType;
   isLoading: boolean;
+  isLogout: boolean;
```

Comment:
I think I would call it : isLoginOut

Reply:
Very Nice :)
Just Waiting for that renaming !

Reply:
`isLoggingOut` :)

Reply:
`logout` is not a valid english word, `disconnect` would be more accurate, isDisconnecting/isDisconnected would be 100% accurate